common/src/service/CommonConstants.cs

```diff
@@ -131,93 +131,5 @@ internal static class CommonConstants
         public const string HttpErrorCodeName = "iothub-errorcode";
 
         public static readonly string[] IotHubAadTokenScopes = new string[] { "https://iothubs.azure.net/.default" };
-
-        //Service Analytics related
-        public static class ServiceAnalytics
```

Comment:
My guess is that these constants are things customers might reference and use when calling our other APIs, which would explain why they don't have any references.

Are we sure we want to remove them? @abhipsaMisra or @timtay-microsoft, any insight here?

Reply:
Never mind; class is internal.